flipt-client-go/models.go

```diff
@@ -7,6 +7,12 @@ type evaluationRequest struct {
 	Context      map[string]string `json:"context"`
 }
 
+type InputEvaluationRequest struct {
```

Comment:
```suggestion
type EvaluationRequest struct {
```

Reply:
wdyt about just EvaluationRequest for all these?

Reply:
@markphelps Yeah I didnt want to expose the fact that you can input a namespace. But there is a cleaner way to do this. Pushed up another commit.